### Rescue

An exception in an unrescued section is logged, and the test terminates.

@[ruby](example.rb)

@[xml](log.xml)
